main/models/Message.nodel.js

```diff
@@ -0,0 +1,29 @@
+const { Schema, model } = require("mongoose");
+
+const messageSchema = new Schema({
+    destination: {
+        type: String,
+        require: [true, { message: "destination is require" }]
+
+    },
+    message: {
+        type: String,
+        require: [true, { message: "message is require" }]
+
+    },
+    number: {
+        type: Number,
+        require: [true, { message: "number is require" }]
+
+    },
+    status: {
+        type: String,
+        enum: ["PENDANT", "CONFIRMED", "REJECTED"],
+        default: "PENDANT"
```

Comment:
Why the status by default is PENDANT? 
Could you please explain, I think there might be an issue here.

Reply:
Every message starts with status: "PENDANT" and if it evrything goos well it is been undpadet to "CONFIRMED",
but if anything goes wrong and the message is not sent I save it in the database as "REJECTED"